### ISSUE TITLE
formatting of INN logo/credit in footer

### ISSUE THREAD

**Issue by aschweigert** [INN/Largo#761]
this is a little large and too prominent, particularly for sites that have footers with less stuff in it (midwest energy brought it up), a few ideas:

- maybe reword the INN/largo/WP credit?
- logo smaller, maybe grayscale?
- too much space between the logo and "member since" line so these look a little disjointed
- should the logo link to the INN homepage? or maybe the member directory instead?
- non-footer option for credit and language to use on an about page, etc. (maybe?)

**Comment by kaeti** [comment]
![footer_larger](https://cloud.githubusercontent.com/assets/884444/8531948/72e3c2b4-23f0-11e5-83e8-d3a40d474663.png)

![footer_small](https://cloud.githubusercontent.com/assets/884444/8531950/72f24fbe-23f0-11e5-8d65-13ea5adcc68e.png)

![footer_similar](https://cloud.githubusercontent.com/assets/884444/8531949/72e7b23e-23f0-11e5-8947-5681e90929c9.png)

Attaching some options. Thinking about making format of our credit more consistent and streamlined led me to try some rearrangements of the footer (also included an option with a layout that stays closer to current styles). 

— I think it would be good to provide at least two config options: one for link-heavy footer and one for a simpler style (a la SE Energy).
— Many people are linking the privacy policy/TOS in the main footer menus and secondary menu. I think we should phase out the secondary/bottom TOS menu.
— Let's put the INN/Largo site credit and the logo together and get rid of the "member since" line.

**Comment by aschweigert** [comment]
Current footer options dialogue for reference:

<img width="775" alt="footer-options" src="https://cloud.githubusercontent.com/assets/490703/8532367/b362f480-23fb-11e5-8615-b274d72f420f.png">

**Comment by aschweigert** [comment]
I like the idea of having an additional option for a more minimal footer for sites that don't have a lot of stuff they want to put down here. I think that might be *in addition to* the current options though. 

I wonder if maybe there's another option that needs to be added here even for the link-heavy one because if you want to call more attention to one column that doesn't really fit well with any of the existing patterns. 

Right now we have:
span3 | span6 | span3
span4 | span4 | span4
span3 | span3 | span3

but I think the mockup you have here would probably be...span2 | span2 | span2 | span6 (or something)

I like giving sites the flexibility to not just use the footer for links but I also think there's probably room to simplify. 

Some other random thoughts:
- Keeping the privacy/TOS with the copyright line makes more sense to me (but I might be able to be swayed).
- I like sometimes having search in the footer (although that's less of an issue for sites that have it in the sticky nav (same deal with "return to top" if the navigation is already present)
- I generally like the grouping of the social/copyright on the left and the INN/credit stuff on the right
- What does this look like for non-INN members? (if the INN logo is not present?) and also, are you sure the logo absent context (i.e. - "member of") is going to make sense to people?

**Comment by kaeti** [comment]
Yeah, I think we could stick with current formats (and add one or two more options), but add the credit info along the bottom for all of the options.

<img width="505" alt="screen shot 2015-07-07 at 9 54 33 am" src="https://cloud.githubusercontent.com/assets/884444/8549395/2a3feda8-2490-11e5-934b-a7c5d3d2ebfc.png">

One that just had space for centered links (minimal).
<img width="504" alt="screen shot 2015-07-07 at 9 54 41 am" src="https://cloud.githubusercontent.com/assets/884444/8549403/332d53ba-2490-11e5-9db7-5211f145b28b.png">

I'm fine with keeping Terms/Privacy in a separate menu, it just annoys me to see them repeated in multiple places in the footer. Also I would simplify the look and put them inline with the copyright notice.

<img width="724" alt="screen shot 2015-07-07 at 9 53 44 am" src="https://cloud.githubusercontent.com/assets/884444/8549438/5e2e1c8e-2490-11e5-94cf-cb300b5e1ada.png">

For non-members, we could easily omit the logo and just have credit text. But we could also leave it for people willing to keep it there. I don't think it's confusing because the logo has the context of "This site built with Project Largo from INN" and it's the INN logo. We could spell out "the Institute for Nonprofit News" in the credit line to make the connection more explicit.

**Comment by aschweigert** [comment]
Yeah, the people use menus is really frustrating. I wonder if there's maybe a way we can be more prescriptive about what should go where. We tried to include that in the docs but I don't think that goes far enough in capturing how menus should be used (just essentially, that they exist).

I think this is enough for me to hack on for a bit and see what I can come up with.

One other wrinkle is that some sites have changed the background color of the footer (for example: http://ctviewpoints.org/) so we probably need a way to reverse out the logo too (or come up with something that would work on either a light or a dark bg)

**Comment by kaeti** [comment]
Yeah, there should be options for light and dark colored footers. For lighter bgs, we can use the standard INN logo. 

![inn_logo_blue_final](https://cloud.githubusercontent.com/assets/884444/8550018/94a7ee9a-2493-11e5-8def-a2c48c1ec50b.png)

**Comment by benlk** [comment]
Just so I'm clear on what needs to be done here:

- [ ] Filter or theme option or constant allowing the INN logo to be swapped to the dark-text logo
- [ ] Footer social icons align left at desktop
- [ ] INN logo aligns right at desktop
- [ ] boilerplate copyright info aligns left at desktop
- [ ] footer bottom menu items appended to end of boilerplate copyright info
- [ ] "This site built with Project Largo from INN and proudly powered by WordPress." aligns right under INN logo at desktop.
- [ ] new footer option, span12, everything centered, like on Southeast Energy News, with one widget area. Menus in this area are initially horizontally centered, with large spaces between the entries, but collapse to a vertical column of centered items.
- [ ] new footer option, ( span7.row-fluid(span4 span4 span4) span5)), with four widget areas. 
- [ ] check for conflicts with all child themes.

How does the new footer boilerplate collapse?

**Comment by aschweigert** [comment]
That seems right. I do want to really think hard about whether we need ALL of these footer options or whether, perhaps one (or more) would not be active/available by default (that is, let's think about the default and the setup experience and make sure we're not making it harder than it needs to be).

As far as how it collapses, the INN stuff should be at the bottom so I'd say the stuff on the left stays (under the widget areas) and the INN stuff at the right folds under that.

**Comment by benlk** [comment]
How do we want to handle intermediate widths? The "This site built with..." footer credit is going to have to wrap at some width.

<img width="1109" alt="screen shot 2015-08-03 at 7 43 35 pm" src="https://cloud.githubusercontent.com/assets/1754187/9050109/0731887e-3a18-11e5-8ce2-891efea13a2d.png">

**Comment by aschweigert** [comment]
- [ ] center of the social icons should be aligned with the center of the INN logo
- [ ] social icon styles should match mockup above (and be consistent with the header and anywhere else they appear)
- [ ] INN credit should read "Built with the [Largo WordPress theme](http://largoproject.org) from the [Institute for Nonprofit News](http://inn.org)."

**Comment by aschweigert** [comment]
- [ ] use span6/span6 for two equal width columns
- [ ] copyright text and INN credit should also be vertically aligned
- [ ] fixed size for the INN logo (smaller is better)
- [ ] stack the two columns, make full width and center elements at an appropriate breakpoint (whenever it starts to look funky or things are colliding)

**Comment by benlk** [comment]
Here's some span6/span6 screenshots:

414px wide:

<img width="414" alt="screen shot 2015-08-04 at 11 33 01 am" src="https://cloud.githubusercontent.com/assets/1754187/9064874/9d4c42d4-3a9c-11e5-8988-c528f1462156.png">


~800px wide: 

<img width="804" alt="screen shot 2015-08-04 at 11 31 08 am" src="https://cloud.githubusercontent.com/assets/1754187/9064829/6120ada4-3a9c-11e5-9ee9-cf4c90437f13.png">

~1300px wide: 
<img width="1394" alt="screen shot 2015-08-04 at 11 31 50 am" src="https://cloud.githubusercontent.com/assets/1754187/9064847/74454c3c-3a9c-11e5-9f80-533b24c1f8d4.png">

**Comment by benlk** [comment]
With .span6/.span6, the footer credit starts to wrap off one line at 945px, so I'm setting the breakpoint for the boilerplate to 969px to match a bunch of other breakpoints.

<img width="884" alt="screen shot 2015-08-04 at 3 34 21 pm" src="https://cloud.githubusercontent.com/assets/1754187/9070566/68e6f828-3abe-11e5-9043-e7053280c603.png">
<img width="1008" alt="screen shot 2015-08-04 at 3 35 26 pm" src="https://cloud.githubusercontent.com/assets/1754187/9070579/7bcde1e0-3abe-11e5-9c34-e53d5891bae9.png">

And when `INN_MEMBER` is false:

<img width="1011" alt="screen shot 2015-08-04 at 3 39 38 pm" src="https://cloud.githubusercontent.com/assets/1754187/9070684/1143e09e-3abf-11e5-9a98-3bac1aba8e55.png">

<img width="835" alt="screen shot 2015-08-04 at 3 40 40 pm" src="https://cloud.githubusercontent.com/assets/1754187/9070711/3488793e-3abf-11e5-8d9c-fed945ad1275.png">

**Comment by aschweigert** [comment]
that seems ok to me, maybe a bit less space under the social icons. @kaeti?

**Comment by kaeti** [comment]
Yeah, agree. There could be a bit less space under the logo and the social icons.

**Comment by benlk** [comment]
The spacing I was using here was matching the brand guidelines of one logo square's worth of space. Because of the descender on the `p`, there's about 4px too much space on the bottom of the image. Is that enough to remove, or should I remove some more and eat into the actual brand spacing on the bottom of the image?

    300px*25px/1669px ~= 4px

**Comment by aschweigert** [comment]
the logo is also still too big, imo

**Comment by kaeti** [comment]
I think we can fudge the brand guidelines for the footer. Leaving that much clear space just doesn't make sense in this context. I don't think you need to remove too much, but we can definitely tighten it up a bit more.

**Comment by benlk** [comment]
In terms of how much space is there now, how much space should be removed? a quarter? half?

The logo above is 300px wide. Without changing the bottom padding, here's a couple smaller sizes

250px:

<img width="1155" alt="screen shot 2015-08-04 at 4 01 26 pm" src="https://cloud.githubusercontent.com/assets/1754187/9071219/204b58a8-3ac2-11e5-84f3-5b1dd263049a.png">

200px:

<img width="1161" alt="screen shot 2015-08-04 at 4 02 27 pm" src="https://cloud.githubusercontent.com/assets/1754187/9071238/3ecbdef6-3ac2-11e5-8995-99bf38dd2a5b.png">

**Comment by kaeti** [comment]
I'd say quarter for the logo (post a screenshot to review). The social icons seem to have more padding underneath, so those we might want half the current space below.

As for logo size, both 250 and 200 look OK. In my mockups, I used a 200px wide logo.

**Comment by benlk** [comment]
<img width="1048" alt="screen shot 2015-08-04 at 4 38 48 pm" src="https://cloud.githubusercontent.com/assets/1754187/9072114/52987eb2-3ac7-11e5-9c2a-185c2ac76371.png">

<img width="900" alt="screen shot 2015-08-04 at 4 39 11 pm" src="https://cloud.githubusercontent.com/assets/1754187/9072124/635372c0-3ac7-11e5-9cfc-9579772cd9df.png">


Changes here: 

- icons approximating mocks, padding 5px instead of 10px.
- logo decreases to 200px wide
- bottom padding on logo decreases by one quarter of a square
- bottom padding on icons decreases by half a square, padding top gains one quarter of a square

**Comment by aschweigert** [comment]
i dig it

**Comment by kaeti** [comment]
looks good!

**Comment by benlk** [comment]
:anchor:

**Comment by benlk** [comment]
> should the logo link to the INN homepage? or maybe the member directory instead?

Is there an answer for this? It currently links to inn.org's homepage.

**Comment by aschweigert** [comment]
homepage is fine
